Backpack/src/main/res/layout/view_bpk_calendar_header.xml

```diff
@@ -1,17 +1,46 @@
 <?xml version="1.0" encoding="utf-8"?>
-<net.skyscanner.backpack.text.BpkText xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto"
   xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
   android:layout_height="wrap_content"
-  android:accessibilityHeading="true"
-  android:maxLines="1"
-  android:textColor="@color/bpkTextPrimary"
-  android:paddingStart="@dimen/bpkSpacingBase"
-  android:paddingEnd="@dimen/bpkSpacingBase"
-  android:paddingTop="@dimen/bpkSpacingLg"
-  android:paddingBottom="@dimen/bpkSpacingLg"
-  android:singleLine="true"
-  android:gravity="start|center_vertical"
-  app:textStyle="heading4"
-  tools:targetApi="p" />
+  android:clipToPadding="false"
+  android:clipChildren="false"
+  android:paddingTop="@dimen/bpkSpacingMd"
```

Comment:
it looks like the padding changed here as we have padding md + lg from the text view now, which explains the difference in the screenshots

Reply:
Nice one @marianeum, that answers one of my questions